app/assets/stylesheets/components/_action-link.scss

```diff
@@ -1,5 +1,6 @@
 .app-c-action-link {
   display: table;
+  margin-bottom: govuk-spacing(2);
```

Comment:
If you make this change here it affects every instance of this component. So... no. Better to put some equivalent top spacing on the `covid__list-item-description` you're adding below.

Reply:
http://govuk-collec-featured-d-9oli5t.herokuapp.com/coronavirus